cognito/mock/cognito_stub.go

```diff
@@ -647,6 +648,20 @@ func (m *CognitoIdentityProviderClientStub) AdminListGroupsForUser(
 		Groups:    newGroups,
 		NextToken: &nextTokenNil,
 	}, nil
+}
+
+func (m *CognitoIdentityProviderClientStub) ListGroups(input *cognitoidentityprovider.ListGroupsInput) (*cognitoidentityprovider.ListGroupsOutput, error) {
+	fmt.Println(*input)
```

Comment:
I assume this was adding for debugging purposes? If so remove it before committing or replace with a log call